components/EditEventForm.js

```diff
@@ -80,15 +80,30 @@ class EditEventForm extends React.Component {
         event['endsAt'] = value;
       }
     }
+    let slugChanged = this.state.slugChanged;
+    if (fieldname === 'slug') {
+      slugChanged = true;
+    }
+    if (!slugChanged && fieldname === 'name') {
+      event['slug'] = event['name'].length
+        ? `${slugify(event['name'])}-${this.state.event.parentCollective.id}ev`.toLowerCase()
```

Comment:
`slugify` already set string to lowercase

```suggestion
        ? `${slugify(event['name'])}-${this.state.event.parentCollective.id}ev`
```